src/main/java/rps/CommandLinePrompt.java

```diff
@@ -65,7 +83,7 @@ private String optionallyAddNewLineCharacter(String optionsForDisplay, int gestu
     }
 
     private boolean lastLine(int gestureNumber) {
-        return gestureNumber < Gesture.values().length - 1;
+        return gestureNumber < getGestures().size();
```

Comment:
instead of `getGestures()` it would read nicer if it was just `gestures()`. At the same time, if you want to know the number of gestures available (or in game) you could add a method for that:

```
public int numberOfGesturesInGame() {
  return gestures().size();
}
```